linkerd/app/outbound/src/http/endpoint.rs

```diff
@@ -132,17 +143,66 @@ impl errors::HttpRescue<Error> for ClientRescue {
     }
 }
 
+// === impl Connect ===
+
+impl<T> svc::Param<Option<SessionProtocol>> for Connect<T> {
+    fn param(&self) -> Option<SessionProtocol> {
+        match self.version {
+            http::Version::Http1 => Some(SessionProtocol::Http1),
+            http::Version::H2 => Some(SessionProtocol::Http2),
+        }
+    }
+}
+
+impl<T: svc::Param<Remote<ServerAddr>>> svc::Param<Remote<ServerAddr>> for Connect<T> {
+    fn param(&self) -> Remote<ServerAddr> {
+        self.inner.param()
+    }
+}
+
+impl<T: svc::Param<tls::ConditionalClientTls>> svc::Param<tls::ConditionalClientTls>
+    for Connect<T>
+{
+    fn param(&self) -> tls::ConditionalClientTls {
+        self.inner.param()
+    }
+}
+
+impl<T: svc::Param<Option<opaque_transport::PortOverride>>>
+    svc::Param<Option<opaque_transport::PortOverride>> for Connect<T>
+{
+    fn param(&self) -> Option<opaque_transport::PortOverride> {
+        self.inner.param()
+    }
+}
+
+impl<T: svc::Param<Option<http::AuthorityOverride>>> svc::Param<Option<http::AuthorityOverride>>
+    for Connect<T>
+{
+    fn param(&self) -> Option<http::AuthorityOverride> {
+        self.inner.param()
+    }
+}
+
+impl<T: svc::Param<transport::labels::Key>> svc::Param<transport::labels::Key> for Connect<T> {
+    fn param(&self) -> transport::labels::Key {
+        self.inner.param()
+    }
```

Comment:
nit: should these `param` methods all be inlined?
```suggestion
    #[inline]
    fn param(&self) -> Remote<ServerAddr> {
        self.inner.param()
    }
}

impl<T: svc::Param<tls::ConditionalClientTls>> svc::Param<tls::ConditionalClientTls>
    for Connect<T>
{
    #[inline]
    fn param(&self) -> tls::ConditionalClientTls {
        self.inner.param()
    }
}

impl<T: svc::Param<Option<opaque_transport::PortOverride>>>
    svc::Param<Option<opaque_transport::PortOverride>> for Connect<T>
{
    #[inline]
    fn param(&self) -> Option<opaque_transport::PortOverride> {
        self.inner.param()
    }
}

impl<T: svc::Param<Option<http::AuthorityOverride>>> svc::Param<Option<http::AuthorityOverride>>
    for Connect<T>
{
    #[inline]
    fn param(&self) -> Option<http::AuthorityOverride> {
        self.inner.param()
    }
}

impl<T: svc::Param<transport::labels::Key>> svc::Param<transport::labels::Key> for Connect<T> {
    #[inline]
    fn param(&self) -> transport::labels::Key {
        self.inner.param()
    }
```